package.json

```diff
@@ -488,6 +520,10 @@
                                                         "type": "boolean",
                                                         "default": true,
                                                         "description": "Enable the inclusion of Defmt location information in the RTT output for `dataFormat=Defmt`."
+                                                    },
+                                                    "defmtLogFormat": {
```

Comment:
@bugadani  The `probe-rs` CLI, uses 
```
      --log-format <LOG_FORMAT>
          The default format string to use for decoding defmt logs
```

Does it make sense for us to be consistent in the naming of these options?

Reply:
Depends on where https://github.com/probe-rs/probe-rs/pull/2442 ends up at, I'm not opposed.